Makefile

```diff
@@ -0,0 +1,7 @@
+VERSION="v`cat package.json | grep '"version":' | cut -d':' -f2 | sed -e 's/[",[:space:]]//g'`"
```

Comment:
`=` is lazy calling in Makefile. It isn't necessary lazy calling here, so use `:=`.
see: http://qiita.com/takuyachallenge/items/c73b1d2db3761b931109#%E5%8D%98%E7%B4%94%E5%B1%95%E9%96%8B%E5%A4%89%E6%95%B0

Reply:
The code below seems better.

```
VERSION="v`node -pe 'require('./package.json').version`"
```

Reply:
Using `node -e` seems much better! Thanks.

Fixed both reviewed points at https://github.com/bitjourney/react-simple-image/pull/24/commits/93700ff5d0ca93ac23856324805050d7c1ff941e